UI/GameInfoCache.cpp

```diff
@@ -405,6 +405,10 @@ class GameInfoWorkItem : public PrioritizedWorkQueueItem {
 					std::lock_guard<std::mutex> lock(info_->lock);
 					info_->paramSFO.ReadSFO(sfoData);
 					info_->ParseParamSFO();
+					if (info_->id.empty() || info_->id == "UCJS10041") {
+						info_->id = g_paramSFO.generateFakeID(gamePath_);
```

Comment:
Forgot to note, but I added the id_version.
 I didn't saw it used anywhere, but it doesn't hurt, I think it's currently only used for savestates maybe also for reporting, but we this pr doesn't mess with those.